Makefile

```diff
@@ -72,7 +72,7 @@ clean:
 
 # Run configured linters
 lint:
-	golangci-lint run
+	golangci-lint run --timeout 5m0s
```

Comment:
The `default` value is 1m0s.  My latest run of this PR took 1m38s.  I'm setting it to 5 minutes for a good buffer.